ic-os/components/boundary-guestos/opt/ic/bin/setup-ic-gateway.sh

```diff
@@ -135,8 +122,9 @@ POLICY_DENYLIST_SEED="${RUN_DIR}/denylist.json"
 DOMAIN_CANISTER_ALIAS="identity:rdmx6-jaaaa-aaaaa-aaadq-cai,nns:qoctq-giaaa-aaaaa-aaaea-cai"
 DOMAIN_CUSTOM_PROVIDER="https://boundary.caffeine.ai/subdomains,https://beta.boundary.caffeine.ai/subdomains"
 GEOIP_DB="${RUN_DIR}/GeoLite2-Country.mmdb"
-IC_URL="http://127.0.0.1:9000"
+IC_URL="https://bc1-dll02.blockchaindevlabs.com,https://dll02.sg2.icp.162.technology,https://br1-dll01.aviatelabs.co,https://ic0.app"
```

Comment:
out of curiosity, where do those URLs come from?

Reply:
These are the domains of the API boundary nodes. You can get them by making a read state request for the path `/api_boundary_nodes/` (see here for [more information](https://internetcomputer.org/docs/current/references/ic-interface-spec/#http-read-state)). These domains are used to bootstrap the communication of the HTTP gateway with the API boundary nodes. One domain would be good enough.